worker/src/main/java/org/apache/celeborn/service/deploy/worker/storage/MapPartitionFileWriter.java

```diff
@@ -199,6 +199,7 @@ public synchronized void destroy(IOException ioException) {
   public void pushDataHandShake(int numReducePartitions, int bufferSize) {
     this.numReducePartitions = numReducePartitions;
     fileInfo.setBufferSize(bufferSize);
+    fileInfo.setNumReducerPartitions(numReducePartitions);
```

Comment:
"fileInfo.setNumReducerPartitions(numReducePartitions);" will be added by another pr